From cd575a84f10d90759e069f5f2e6092251a1b84bd Mon Sep 17 00:00:00 2001
From: Sven Neumann <s.neumann@raumfeld.com>
Date: Fri, 23 Sep 2011 10:00:02 +0200
Subject: [PATCH] libdirect: remove include of <linux/config.h>

That header has been removed from the Linux kernel with 2.6.19
and it should not have been needed anyway.
---
 lib/direct/ppcasm_memcpy_cachable.S |    2 --
 1 file changed, 2 deletions(-)

diff --git a/lib/direct/ppcasm_memcpy_cachable.S b/lib/direct/ppcasm_memcpy_cachable.S
index 920dea2..c1ef4fa 100644
--- a/lib/direct/ppcasm_memcpy_cachable.S
+++ b/lib/direct/ppcasm_memcpy_cachable.S
@@ -34,8 +34,6 @@
 
 #define __ASSEMBLY__
 
-#include <linux/config.h>
-
 #if defined(CONFIG_8xx) || defined(CONFIG_403GCX)
 #define L1_CACHE_LINE_SIZE       16
 #define LG_L1_CACHE_LINE_SIZE     4 
-- 
1.7.9.5

